fml/time/time_recorder.cc

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/fml/time/time_recorder.h"
+#include "flutter/fml/logging.h"
+
+namespace fml {
+
+TimeRecorder::TimeRecorder() : initialized_(false) {}
```

Comment:
Consider an `std::optional<fml::TimePoint>` instead. That neatly handles the same concerns handled by the `initialized_` flag.